code/_onclick/click.dm

```diff
@@ -492,9 +492,10 @@
 /* MouseWheelOn */
 
 /mob/proc/MouseWheelOn(atom/A, delta_x, delta_y, params)
-	return
+	SEND_SIGNAL(src, COMSIG_MOB_MOUSE_SCROLL_ON, A, delta_x, delta_y, params)
 
-/mob/dead/observer/MouseWheelOn(atom/A, delta_x, delta_y, params)
+/mob/dead/observer/proc/mouse_wheeled(atom/A, delta_x, delta_y, params)
```

Comment:
doesn't really matter but any specific reason this was changed to use the signal?

Reply:
Wouldn't it override the /mob/MouseWheelOn base proc? Shouldn't we have it call just the signal and from there we recieve messages depending on what we need?

Reply:
I mean, I guess, but you can do a supercall and it's the same, plus no need to use signal. But it works as is